deploy/config/keter.yaml

```diff
@@ -1,8 +1,9 @@
 stanzas:
-  - type: webapp
+  - type: background
```

Comment:
Thanks for pointing it out, @cdepillabout .
I created an issue #94  about it.

I felt the ensureAlive mechanism is evil because it can cause server down though one of the role of `keter` is stable server hosting.